### PR TITLE
rclone: update to 1.57.0

### PR DESCRIPTION
**Pull request by 4ricci**
- Add mount helper

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by ericonr** [comment]
Tests are failing on i686, I'm trying again/

**Comment by 4ricci** [comment]
> Tests are failing on i686, I'm trying again/

They fail locally too. Maybe tomorrow I will investigate better.

**Comment by 4ricci** [comment]
Two tests fail: [`backend/cache`](https://github.com/rclone/rclone/issues/5784) and [`cmd/serve/ftp`.](https://github.com/rclone/rclone/issues/5783) The first is a deprecated feature anyway; the second just times out, it passes if the time limit is increased.
I think this can be merged.

File: srcpkgs/rclone/template [hunk 1]
```diff
@@ -32,6 +32,7 @@ do_check() {
 }
 
 post_install() {
+	ln -sf rclone ${DESTDIR}/usr/bin/mount.rclone
```
Comment (ericonr) [inline srcpkgs/rclone/template:35]: Hm, it should have been `ln -s rclone ...` instead.

**Pull request merged**